kubejs/assets/ftbquests/lang/pl_pl.json

```diff
@@ -0,0 +1,130 @@
+"chapter.1.quest.54.subtitle": "CHCĘ WIĘCEJ MIEJSCA NA PRZECHOWYWANIE",
+"chapter.1.quest.54.task.1.title": "Zrozumiałem.",
+"chapter.1.quest.54.title": "Ulepszenia przechowywania",
+"chapter.1.quest.55.description.1": "Stwórz jeden pakiet, używając wymaganych składników z poprzedniego rozdziału.\n\n\nMożesz mieszać i łączyć, a także używać materiałów dekoracyjnych innych niż drewno. Odkryj swoje opcje w REI!",
+"chapter.1.quest.55.subtitle": "Tropikalna dżungla inwentarza.",
+"chapter.1.quest.55.task.1.title": "Dowolny #pakiet:banned_from_package_maker",
+"chapter.1.quest.55.title": "Spakowany",
+"chapter.1.quest.56.description.1": "Winifikacja podąża tym samym szlakiem co piwo, z produkcją odbywającą się za pomocą bloku unikatowego\n\n\nWina produkowane będą miały aktualny rok Minecraft wydrukowany na nich. Im dłużej będą leżały bez picia, tym ich efekty mogą stać się potężniejsze...",
+"chapter.1.quest.56.subtitle": "Wygląda na to, że picie tego jest bardziej wyrafinowane...",
+"chapter.1.quest.56.title": "Wybrany szlif...",
+"chapter.1.quest.57.description.1": "Cydr jabłkowy - 2x sok jabłkowy, 1x kosz\nWino jabłkowe - 2x sok jabłkowy, 1x butelka miodu\nCydr kelp - 1x sok jabłkowy, 2x kelp",
+"chapter.1.quest.57.subtitle": "Przepisy w środku!",
+"chapter.1.quest.57.title": "Sok różnych jabłek.",
+"chapter.1.quest.58.description.1": "Wino Bolvar - 2x czerwony, 1x butelka miodu, 1x wiśnia\nWino Chenet - 2x czerwony, 2x słodka jagoda\nWino Noir - 2x czerwony\nKról Danis - 2x czerwony, 2x butelka miodu\nWino Wiśnia - 2x czerwony, 2x wiśnia",
+"chapter.1.quest.58.subtitle": "Przepisy w środku!",
+"chapter.1.quest.58.title": "Sok różnych winogron czerwonych.",
+"chapter.1.quest.59.description.1": "Wino Solaris - 2x biały, 1x słodka jagoda, 1x butelka miodu\nWino Clark - 2x biały, 2x słodka jagoda\nWino Mellohi - 2x biały, 2x butelka miodu",
+"chapter.1.quest.59.subtitle": "Przepisy w środku!",
+"chapter.1.quest.59.title": "Sok różnych winogron białych.",
+"chapter.1.quest.6.description.1": "Jeśli nie lubisz Tinkers' Construct, narzędzia stylu vanilla mogą być wytworzone z lingotów miedzi, aby ci pomóc.",
+"chapter.1.quest.6.subtitle": "Opcje, ludzie, opcje",
+"chapter.1.quest.6.title": "Żałuję starego sposobu",
+"chapter.1.quest.60.description.1": "Sok jabłkowy - 2x puree jabłkowy",
+"chapter.1.quest.60.subtitle": "Przepis w środku!",
+"chapter.1.quest.60.title": "Zgnieść tego kota",
+"chapter.1.quest.61.subtitle": "Umieść kilka winogron w tych beczkach i idź! Zbierz sok, używając pustych butelek wina.",
+"chapter.1.quest.61.title": "Zobacz, jak te tłoczki idą",
+"chapter.1.quest.62.subtitle": "Uwaga: Winiarnia nie ma wbudowanego wsparcia REI!",
+"chapter.1.quest.62.title": "Kuchnia winiarni",
+"chapter.1.quest.63.title": "Minerały Overworlda",
```

Comment:
No idea how Overworld is officialy translated in Minecraft, despite playing Minecraft in Polish for a long time.